src/msr_data_skx.cpp

```diff
@@ -96,7 +96,8 @@ namespace geopm
                     "units":     "hertz",
                     "scalar":    1e8,
                     "writeable": false,
-                    "aggregation": "average"
+                    "aggregation": "average",
+                    "description": "The value associated with the current operating frequency of the CPU."
```

Comment:
The current operating frequency of the CPU based on the measurement of the status register.